uqcsbot/base.py

```diff
@@ -260,9 +263,17 @@ def __init__(self, bot: UQCSBot):
 
     def __getattr__(self, name):
         attr = getattr(self.bot, name)
-        if not callable(attr) or not asyncio.iscoroutinefunction(attr):
+        if not callable(attr) or asyncio.iscoroutinefunction(attr):
```

Comment:
what was the reason for removing the `not` here?

Reply:
Fixing the behaviour - if it's a coroutine function we want to return it straight up because it's already async